source/_partials/content/https-requirements.html

```diff
@@ -0,0 +1,10 @@
+<h3>Requirements for Contiguously Managed HTTPS</h3>
+<ul>
```

Comment:
Should we include links to additional information for each bullet? Otherwise we're assuming folks know how to implement or evaluate requirements. Any tips for what links to add @ari-gold @kf6nux ? If ya'll think it's fine as is, we can ship.

Reply:
How about: "Requirements for automated certificate renewal" ?

Reply:
@ari-gold do you think we're okay to ship without additional links/context? Or should we require those in this PR?